pages/2_📖_Build_App.py

```diff
@@ -79,5 +75,11 @@
 ### Complex workflow using TOPP tools
 
 This template app features a module in `src/workflow` that allows for complex and long workflows to be built very efficiently. Check out the `TOPP Workflow Framework` page for more information (on the *sidebar*).
-""")
 
+## How to package everything for window executables
```

Comment:
Could be a separate page in the documentation, just for Windows executables?!
Import markdown files and display content directly in streamlit app [like here?](https://github.com/OpenMS/streamlit-template/blob/main/pages/4_%F0%9F%93%96_Deployment.py)

Reply:
yes, sure it is also possible.

Reply:
@axelwalter 
![Screenshot 2024-05-08 145106](https://github.com/OpenMS/streamlit-template/assets/93129328/3314802b-73af-4f2a-b88e-f7c994bdf57a)
any suggestion here?